addons/l10n_id/models/account_move.py

```diff
@@ -0,0 +1,158 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+import pytz
+
+from odoo import api, fields, models, _
+
+
+class AccountMove(models.Model):
+    _inherit = 'account.move'
+
+    # Used to fetch the QR payment status.
+    # List of dicts with the format: [{
+    #     'qris_invoice_id': '13246',
+    #     'qris_amount': 0.0,
+    #     'qris_creation_datetime': '2024-02-27 03:00:00',
+    # }]
+    l10n_id_qris_invoice_details = fields.Json(
+        string="QRIS Transaction Number",
+        help="Transaction Number stored.",
+        readonly=True,
+        export_string_translation=False,
+    )
+    # Used to return the same QR code if still valid.
+    l10n_id_latest_qris_content = fields.Char(
```

Comment:
How do you know that it is still valid?
Shouldn't it be computed based on `l10n_id_qris_invoice_details`?

Reply:
In the context where it is used, we check for it.
But there are indeed no guarantees that it will always be.

I guess we can compute it yes, it would make sense :thinking: good idea